docs/tracker_stores.rst

```diff
@@ -94,7 +131,7 @@ MongoTrackerStore
 
                 python -m rasa_core.run --core models/dialogue --endpoints endpoints.yml
 :Parameters:
-    - ``url`` (default: ``mongodb://localhost:27017``): URL of your MongoDB
+    - ``host`` (default: ``mongodb://localhost:27017``): URL of your MongoDB
```

Comment:
You can't just change these parameters for mongo & redis (you didn't change the code, and I also don't think we should).

Reply:
I'm confused, in the code for mongo & redis, the parameter is `host`.  How does `url` get parsed as `host` when we include it in the `endpoints.yml`?